docs/SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Reporting security issues
```

Comment:
You can add a [Hugo front matter](https://gohugo.io/content-management/front-matter/) so that Hugo orders the files in the desired order (with the `weight` parameter). See other markdown files.